osimpipeline/task.py

```diff
@@ -96,29 +105,183 @@ def __init__(self, trial):
         super(TrialTask, self).__init__(trial.subject)
         self.trial = trial
 
+class SetupTask(TrialTask):
+    def __init__(self, tool, trial, 
+                 model_to_adjust=None, 
+                 adjusted_model=None,
+                 gait_cycles='concatenated'):
```

Comment:
If we're going to merge this into master, it might be good to think a little more about the names "gait_cycles," "concatenated," and "separated." We're not actually passing in gait cycles for this parameter; another name would be "gait_cycle_style." We could also use a boolean for now, as you suggested (perhaps "process_cycles_separately," "per_cycle," or "separate_cycles" ).